### PR TITLE
Put Optic Cloud extensions under `openapi` tag

### PR DESCRIPTION
**Pull request by acunniffe**
## 🍗 Description
Users said they were hard to find if they had to scroll to the bottom

## 📚 References
_Links to relevant docs (Notion, Twist, GH issues, etc.), if applicable._

## 👹 QA
_How can other humans verify that this PR is correct?_

### CODE REVIEW

File: projects/optic/src/commands/api/write-to-file.ts [hunk 1]
```diff
@@ -34,23 +39,69 @@ export async function writeYml(
...
+
+      return (
+        cleanedYaml.substring(0, indexOfOpenApiLineNewLine) +
+        (extensionLines.length ? '\n' + extensionLines.join(`\n`) : '') +
+        cleanedYaml.substring(indexOfOpenApiLineNewLine)
+      );
+    } else {
+      throw new Error('openapi: 3.x.x line not found');
+    }
+  } catch (e) {
```
Comment (acunniffe) [inline projects/optic/src/commands/api/write-to-file.ts:96]: wrote this with a pretty robust fallback mechanism...if we get no-ops re: `openapi` is spelled wrong or other weirdness resulting in an invalid patch, it just write it to yaml naivelly.

File: projects/optic/src/commands/api/add.ts [hunk 2]
```diff
@@ -121,6 +121,14 @@ async function promptForStandard(
...
   const existingStandards = await client.getOrgStandards(orgId);
 
+  // if empty, create default without interrupting first time users
+  if (existingStandards.length === 0) {
+    const standard = await client.createOrgStandard(orgId, [
+      { name: 'breaking-changes', config: {} },
+    ]);
+    return standard.slug;
+  }
+
```
Comment (niclim) [inline projects/optic/src/commands/api/add.ts:131]: Is this intended to be part of this PR?

If yes, should we add any logging here to show that a standard has been added?

**Review by niclim — APPROVED** [review]
Do we also need to handle JSON as well?

File: projects/optic/src/commands/api/add.ts (hunk 2 above)
Comment (acunniffe) [reply projects/optic/src/commands/api/add.ts:131]: nope removing this

**Comment by acunniffe** [comment]
> Do we also need to handle JSON as well?

I suppose we should . I'll update

**Pull request merged**